iocage_lib/ioc_common.py

```diff
@@ -1032,3 +1032,32 @@ def parse_package_name(pkg):
         'revision': revision,
         'epoch': epoch,
     }
+
+
+def get_host_gateways():
+    netifaces_af_mapping = {
+        'Internet': netifaces.AF_INET,
+        'Internet6': netifaces.AF_INET6
+    }
+
+    output = su.run(['netstat', '-r', '-n', '--libxo', 'json'],
+                    stdout=su.PIPE)
+    route_families = (json.loads(output.stdout)
+                      ['statistics']
+                      ['route-information']
+                      ['route-table']
+                      ['rt-family'])
+
+    gateways = {'default': {}}
```

Comment:
We are only interested in retrieving `default`, do you think perhaps it would be better to map `netifaces.AF_INET\6` directly instead of accessing `default` key always before ?

Reply:
Yeah, you are right. I wanted to keep up the `netifaces` API. But with only one default value being returned this is very artificial.